### ISSUE TITLE
Configuration regression in v1.3.0 due to ef910e7

### ISSUE THREAD

**Issue by sylv-io** [charmbracelet/mods#254]
**Describe the bug**
Since I updated mods to v1.3.0 I get the following error:
```
 ERROR Could not use role
```

I have never used the roles feature in mods before, so I did a quick bisect and was able to determine the commit that causes this error: ef910e7
Since this change, the config key `role` is expected to exist and point to a valid key in `roles`.

However, the config_template.yml introduced both of these now required values in v1.2.2 with commit e48f558. Therefore, any configuration initialized before this version will have this regression.

**To Reproduce**
1. Remove/backup your mods.yml configuration file.
```
mv ~/.config/mods.yml ~/.config/mods.yml.bak
```
2. Checkout a branch before commit e48f558
```
git checkout e48f558~
```
3. Run mods command to create default configuration
```
go run . --no-cache "Say Hello, World!"
```
4. Checkout branch after commit ef910e7
```
git checkout ef910e7
```
5. Run a mods command again
```
go run . --no-cache "Say Hello, World!"
```

**Expected behavior**
The last prompt answers: "Hello, world!"

**Actual behavior**
```

   ERROR  Could not use role  

  role "" does not exist  

exit status 1
```

**Additional context**
A simple fix is to set the `role` and `roles` config keys to optional. The main reason for creating this issue is for anyone who has experienced the same behavior and is looking for the cause.

**Comment by ArthurHeymans** [comment]
:+1: works for me, thanks

**Comment by caarlos0** [comment]
fixed in #255, thanks @sylv-io
